MCDataProducts/inc/GenId.hh

```diff
@@ -44,7 +44,8 @@ namespace mu2e {
       cosmicCRY,  pbarFlat, fromAscii, ExternalRMC, InternalRMC, CeLeadingLog, cosmicCORSIKA, //44
       MuCapProtonGenTool, MuCapDeuteronGenTool, DIOGenTool, MuCapNeutronGenTool, // 48
       MuCapPhotonGenTool, MuCapGammaRayGenTool, CeLeadingLogGenTool, MuplusMichelGenTool,// 52
-      lastEnum //53
+      gammaPairProduction, //53
```

Comment:
For art resampling we use ProcessCode to describe the production of stopped particle daughters.   That is also how we identify particles in FindMCPrimary.  I see that you use that below as well, why do you also need this genId?

Reply:
I found it useful in the dump of SimParticle collection